Tutorials/TurnBasedRPG/source/CombatHUD.hx

```diff
@@ -157,6 +158,7 @@ class CombatHUD extends FlxTypedGroup<FlxSprite>
 	 */
 	public function initCombat(PlayerHealth:Int, E:Enemy):Void
 	{
+		_sprScreen.drawFrame();
```

Comment:
Because it says to my class the target need to `drawFrame()` again, and this call `calcFrame` that clear the custom draw of line 164.